longruntests/longruntests_suite_test.go

```diff
@@ -0,0 +1,404 @@
+package longruntests_test
```

Comment:
Also, can we not use `_test` in package names? Everything in this repo is tests, so I think this suffix is redundant